### ISSUE TITLE
forum thread link for vehicle page

### ISSUE THREAD

**Issue by rusefi** [rusefi/web_backend#57]
each vehicle page should link to one forum thread

we show a list of last 25 threads posted by user for user to select his topic
also we allow user to enter numeric topic ID and validate that this is his topic.

**Comment by andreika-git** [comment]
I've added a new `topic_id` field to `msqur-engines`.
Also, the user 685 already has it set to his vehicle's topic ID, for testing purposes.

**Comment by andreika-git** [comment]
https://github.com/rusefi/msqur/commit/c92dd47838789eee83dad68c01ff4c9e733f110e

**Comment by rusefi** [comment]
we now need to change rusEFI custom phpbb code to have a control to select topic ID for each of user's engines

@ksmola any interest to join the fun? you would probably need a sandbox pbpbb instance

**Comment by rusefillc** [comment]
Status update: we have REO part ready - i.e. REO displays forum link if forum relation is present in the database

We now need the rusEFI forum part of the change - i.e. user control panel should allow to designate any thread started by a user as vehicle thread for any of the user's vehicles

@ksmola can you help here?

**Comment by rusefillc** [comment]
Best cost/benefit ratio across the whole rusEFI universe in my opinion.

**Comment by rusefillc** [comment]
rusEFI phpbb is at https://github.com/rusefi/rusefi/tree/master/misc/www/forum

**Comment by cookiesowns** [comment]
Just a thought, why not just have the user link to the topic they've created on the forums when they add a engine/car in the rusEFI Online portal?

means we wouldn't have to touch the PhpBB side...

Maybe I'm not seeing the whole picture on what the intention of this is other than linking from rusEFI online to a forum thread.

Is the idea that the forum thread will also link back to rusEFI online, and display a list of rusEFI online engines in the users profile on the forum side?

**Comment by rusefillc** [comment]
@cookiesowns you are absolutely right, there are definitely many way to do this, any way would work as long as it works :)

Maybe not touch the "upload" page for the user but give user a control to pick forum thread while he is viewing "vehicle" page?
